tests/apple-exporter/components/test-class-body.php

```diff
@@ -14,6 +14,56 @@
  */
 class Apple_News_Body_Test extends Apple_News_Testcase {
 
+	/**
+	 * A data provider that arguments for `test_dropcap_determination` tests.
+	 *
+	 * @return array An array of arrays representing function arguments.
+	 */
+	public function data_dropcap_determination() {
+		return [
+			// No dropcap -- punctuation first character.
+			[
+				'default-body',
+				100,
+				'yes',
+				<<<HTML
+<p>"Go-Gurt," but to stay.</p>
+HTML
+				,
+			],
+			// No dropcap -- dropcap minimum character requirement not met.
+			[
+				'default-body',
+				100,
+				'no',
+				<<<HTML
+<p>I hope to keep this briefing brief. But briefly, before I begin this brief briefing...</p>
+HTML
+				,
+			],
+			// Dropcap applied -- minimum character opt out.
+			[
+				'dropcapBodyStyle',
+				500,
```

Comment:
This seems wrong - shouldn't it use the dropcap body style only if the character length is _over_ 500?

Reply:
This one is testing the opt out flag.  So ordinarily it would use the dropcap body style only if the character length is over 500, but the `dropcap_minimum_opt_out ` is set to `'yes'`.  So it ignores the `dropcap_minimum` character requirement and applies the dropcap style.

Reply:
Oh, I see, it's testing to make sure it's below the minimum, which would ordinarily cause it to not use the dropcap, but using the opt-out flag it's going to use it anyway. Carry on!